packages/pn-pa-webapp/src/components/NewNotification/PreliminaryInformations.tsx

```diff
@@ -148,11 +153,17 @@ const PreliminaryInformations = ({ notification, onConfirm }: Props) => {
   });
 
   const handleChangeTouched = async (e: ChangeEvent) => {
+    
     formik.handleChange(e);
     await formik.setFieldTouched(e.target.id, true, false);
   };
 
   const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
+    const value = e.target.value;
+    if(value === 'it'){
+      void formik.setFieldValue('additionalLang','');
```

Comment:
This code partially solves the bug. It empties the fields but it doesn't reset the touched state. Use the formik.setValues to set additionalLang and additionalSubject in one function and set the second parameter to false to skip the validation. Use formik.setFieldTouched to set the touched state of additionalSubject.

Reply:
doijne